src/App.vue

```diff
@@ -1,27 +1,9 @@
-<template>
-  <img alt="Vue logo" src="./assets/logo.png">
-  <HelloWorld msg="Welcome to Your Vue.js + TypeScript App"/>
-</template>
-
-<script lang="ts">
-import { defineComponent } from 'vue';
-import HelloWorld from './components/HelloWorld.vue';
+<template><router-view> </router-view></template>
```

Comment:
```suggestion
<template>
  <router-view/>
 </template>
```

Format the template in parent child manner